python/test/test_dynamic_bitset.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+
+from katana.dynamic_bitset import DynamicBitset
+
+SIZE = 50
+
```

Comment:
Your code style is off here. You will want to run `scripts/check_python_format.sh -fix python` and `scripts/check_python_lint.sh -fix python` on your code. (Those commands may not be exactly correct, but they should be close.)

Reply:
done (and I hope that fixed the formatting)